plugins/webp-uploads/picture-element.php

```diff
@@ -75,6 +75,11 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 		return $image;
 	}
 
+	// If jpeg fallback is not available, don't wrap in picture element.
+	if ( ! in_array( 'image/jpeg', $mime_types, true ) ) {
```

Comment:
```suggestion
	if ( ! in_array( $original_file_mime_type, $mime_types, true ) ) {
```
If we use the original upload mime type here, this will continue to work when we add handling for PNG uploads in https://github.com/WordPress/performance/pull/1421

Reply:
@adamsilverstein I have updated the check with the original mime type. Thank you for the suggestion.